### DISCUSSION TITLE
Using the filter_duplicates() function

### DISCUSSION THREAD

**Discussion by sermomon** [r-lidar/lidR#498] [Q&A]
If I have a point with X, Y, Z coordinates and 3 different returns. `filter_duplicates()` removes two of the returns? That is, does the function consider each return as a point with coordinates X, Y, Z or are they a single point? 

I am concerned with applying this function and leaving a single return for each of my X, Y, Z coordinates.

**Comment by Jean-Romain** [comment]
filter_duplicate removes points that have the exact same XYZ coordinates (keep one occurence). Not multiple returns.
